apps/aws-amplify/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "aws-amplify",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@contentful/app-sdk": "^4.17.2",
+    "@contentful/f36-components": "4.41.1",
+    "@contentful/f36-tokens": "4.0.1",
+    "@contentful/react-apps-toolkit": "1.2.15",
+    "contentful-management": "10.35.6",
+    "emotion": "10.0.27",
+    "lodash": "^4.17.21",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "react-scripts": "5.0.1"
+  },
+  "scripts": {
+    "start": "cross-env BROWSER=none react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "create-app-definition": "contentful-app-scripts create-app-definition",
+    "deploy": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id ${DEFINITIONS_ORG_ID} --definition-id YlAVHsxexkcs5PG3INCmc --token ${CONTENTFUL_CMA_TOKEN}",
```

Comment:
Did a sanity check here and it matches and looks good! Just create a ticket to make sure we handle the Samson deployment and take away it's `localhost:3000` reference in the app defintion